### ISSUE TITLE
app does not work with firmware 2.x.x.x

### ISSUE THREAD

**Issue by tonnoz** [WilixLead/iWownController#3]
Hello I tried to download your app from the store but it is not working with the latest firmware of the bracelet. Why? Can I help contribute?

**Comment by tonnoz** [comment]
I think the problem is just to find out the new [Constants ](https://github.com/WilixLead/iWownController/blob/master/Application/src/main/java/ru/wilix/device/geekbracelet/i5/Constants.java) used from the new firmware, right?

**Comment by WilixLead** [comment]
Hello. I can't say about it now. Just I not use my bracelet some time and have old firmware.
I think problem may contains not only in this. iWown in some firmwares change their characteristics in BT stack to another. May be it happends again.
For understand what changed I should decompile latest Zeroner app and make reverse-engenering

**Comment by tonnoz** [comment]
Sure, I have the new apk of the Zeroner app 3.0.x how can decompile/reverrse engineer it?
As mentioned, I can help making it work on the new firmware.

**Comment by WilixLead** [comment]
Yes. But now I have no time for this. Too many people ask me about it :) I am very want to fix this app.
But, primary work for first

**Comment by mohamedchouat** [comment]
hello,
iam working with your lib but i can't find the number of steps plz help :)
